### DISCUSSION TITLE
Mastodon Webhook: What exactly is "Private" and how can I post to Followers only?

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6732] [Q&A] [answered]
I’m just registering a webhook for my mastodon.social account. Works fine so far but I’m unsure what _Message Visibility: Private_ means:

![Auswahl_301](https://github.com/AzuraCast/AzuraCast/assets/3706922/3dfffdd9-0093-40e7-bbea-14debc091f9a)

My Mastodon usually shows these options:

![Auswahl_300](https://github.com/AzuraCast/AzuraCast/assets/3706922/16195c95-176b-4850-95be-d780d82ab37c)

I _assume_ the first two are identical but what is _Private_? Would this be _Followers_ & _Mentioned_ in one or something?

The idea behind: I’d like to offer _followers_ a means to subscribe to DJ and station on/off statuses.

**Comment by BusterNeece** [comment, marked as answer]
@Moonbase59 In their API (and in ActivityPub's spec) "private" is what they now call "Followers Only".

The fourth option, "mentioned people only", is a direct message, which we don't support.

**Reply by Moonbase59** [reply]
Cool, thanks! Exactly what I wanted :-)

**Reply by Moonbase59** [reply]
And thanks for renaming it in the software, this really helps if you come back after a few weeks and forgot…
